internal/providers/terraform/aws/dx_gateway_association.go

```diff
@@ -8,14 +8,21 @@ import (
 
 func GetDXGatewayAssociationRegistryItem() *schema.RegistryItem {
 	return &schema.RegistryItem{
-		Name:  "aws_dx_gateway_association",
-		RFunc: NewDXGatewayAssociation,
+		Name:                "aws_dx_gateway_association",
+		RFunc:               NewDXGatewayAssociation,
+		ReferenceAttributes: []string{"associated_gateway_id"},
```

Comment:
Should we document this feature in the contributing docs? I suspect few of our contributors know about it

Reply:
Yep, it's fine to say that launching a resource is needed, I suspect most infracost contributors will be used to launching with TF